public/js/list-changes-chart-builder.js

```diff
@@ -0,0 +1,83 @@
+var ListChangesChartBuilder = (function() {
```

Comment:
I need to do this for the rest of my files then I believe (`time-spent.js` / `spec` as well)